.github/stale.yml

```diff
@@ -38,7 +31,7 @@ markComment: >
   acceptable!
 
 closeComment: >
-  As I didn't see any updates I'm going to close this due to being stale.
+  As I didn't see any updates in 30 days I'm going to close this.
   Please see the previous comment for more information!
```

Comment:
Maybe add something about when it would be appropriate to reopen (or whom to ping if the person reading the comment is not the original reporter)?

Reply:
Sure!

I think just commenting should be enough, as we have enough repo watchers.

In terms of "when" what were you thinking?